lldb/include/lldb/API/SBDebugger.h

```diff
@@ -426,6 +426,8 @@ class LLDB_API SBDebugger {
 
   SBTypeSynthetic GetSyntheticForType(SBTypeNameSpecifier);
 
+  void ResetStatistics();
```

Comment:
Add a header doc comment for this to document what this will do:
- clear all stats for all modules in all targets
- clear all target breakpoint stats?

Reply:
Comments added. I do not think we need to clear any breakpoint stats because they are not reused across debug sessions by "reuse lldb-dap"